recipes/hdf5-lz4/build.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+PPATH="/usr/local/hdf5/lib/plugin"
+
+mkdir -vp ${PREFIX}/$PPATH
+
+export CXXLAGS="${CFLAGS}"
+export CPPFLAGS="-I${PREFIX}/include"
+export LDFLAGS="-L${PREFIX}/lib"
+
+make -e HDF5_INSTALL="${PREFIX}/$PPATH"
```

Comment:
ah, I was doing wishful API reading drop that last change,  Sorry :disappointed: 

This is telling it where hdf5 is installed, not where it should install it's files.